BackEnd/routes.js

```diff
@@ -209,6 +211,43 @@ router.get('/getTargetLocation', (req, res) => {
 	    res.send(obj.location);
 	}
     });
+
+router.post('/createAlliance', (req, res) => {
+	var jsonRequestBody = req.body;
+
+	var allianceFields = {};
+	allianceFields.allies = jsonRequestBody.allies;
+	allianceFields.targets = jsonRequestBody.targets;
+
+	var alliance = new Alliance(allianceFields);
+	var allianceId = alliance.id;
+	var allies = JSON.parse(jsonRequestBody.allies);
+
+	alliance.save(function(err) {
+		if (err) {
+			console.log("Failed to create alliance - Error: " + err.message);
+			return;
+		}
+	});
+
+	// Update users with their alliance ID
+	for (int i = 0; i < allies.length; i++) {
+		Player.findOneAndUpdate({
```

Comment:
Can we change the target type in Player schema to an array to add the targets of each alliance member to each other member's target list? That would vastly simplify killTarget...